<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Utama</string>
  </property>
  <widget class="QPushButton" name="button_tampil_iklan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Iklan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_kategori">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Kategori</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_kota">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Kota</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_member">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_pembayaran">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Pembayaran</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_pesan">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Form Pesan Inbox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM UTAMA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
